stdlib/public/core/BidirectionalCollection.swift

```diff
@@ -247,8 +249,8 @@ extension BidirectionalCollection where SubSequence == Self {
   ///   elements in the collection.
   ///
   /// - Complexity: O(1) if the collection conforms to
-  ///   `RandomAccessCollection`; otherwise, O(*n*), where *n* is the length
-  ///   of the collection.
+  ///   `RandomAccessCollection`; otherwise, O(*k*), where *k* is the number of
+  ///   elements to remove.
   @inlinable // protocol-only
   public mutating func removeLast(_ n: Int) {
```

Comment:
I don't think we should change the doc in this way without also changing the parameter name to `k`.